src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -131,6 +131,14 @@ public EventHistory buildEvents(CaseData caseData) {
                         }
                         break;
                     }
+                    case PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_COS: {
+                        // this would change in CIV-1620
```

Comment:
is this would change why are we updating this class

Reply:
New state intorduced for not changing current functionality in production and allowing the user event "Notify claim" only when COS is enabled. It was added to allow "notify claim" for COS in "FlowStateAllowedEvent.java"

Reply:
I am going to test it without adding new event to EventhistoryMapper.